ROADMAP.md

```diff
@@ -17,7 +17,6 @@ The goal is to improve adoption and sustainability of the project.
   - Create [Patreon page](https://www.patreon.com/syrusakbary)
```

Comment:
Are we keeping a Patreon page on the ROADMAP?

Reply:
Good catch. Updated accordingly